dist/obs-server.spec

```diff
@@ -664,6 +709,29 @@ usermod -a -G docker obsservicerun
 %ghost /srv/www/obs/api/log/production.log
 %ghost %attr(0640,root,www) %secret_key_file
 
+%{_unitdir}/obs-api-support.target
+%{_unitdir}/obs-clockworkd.service
+%{_unitdir}/obs-delayedjob-queue-consistency_check.service
+%{_unitdir}/obs-delayedjob-queue-default.service
+%{_unitdir}/obs-delayedjob-queue-issuetracking.service
+%{_unitdir}/obs-delayedjob-queue-mailers.service
+%{_unitdir}/obs-delayedjob-queue-project_log_rotate.service
+%{_unitdir}/obs-delayedjob-queue-quick.service
+%{_unitdir}/obs-delayedjob-queue-quick@.service
+%{_unitdir}/obs-delayedjob-queue-releasetracking.service
+%{_unitdir}/obs-sphinx.service
+/usr/sbin/rcobs-api-support
```

Comment:
%{_sbindir}

Reply:
@hennevogel: 

`/usr/sbin` changed to `%{_sbindir}`